napari/layers/image/image.py

```diff
@@ -76,6 +76,15 @@ class Image(IntensityVisualizationMixin, Layer):
         Scale factors for the layer.
     translate : tuple of float
         Translation values for the layer.
+    rotate : float, 3-tuple of float, or n-D array.
+        If a float convert into a 2D rotation matrix using that value as an
+        angle. If 3-tuple convert into a 3D rotation matrix, rolling a yaw,
+        pitch, roll convention. Otherwise assume an nD rotation. Angle
+        conversion are done either using degrees or radians depending on the
+        degrees boolean parameter.
```

Comment:
As above, and really everywhere this docstring lives.

Reply:
fixed